geometry.pyi

```diff
@@ -20,7 +20,7 @@ Coordinate = Union[Tuple[float, float], Sequence[float], Vector2]
 
 _CanBeLine = Union[
     Rect,
-    "Line",
+    Line,
```

Comment:
nope cause the `Line` object is not defined yet

Reply:
I mean idk i just copied what pygame has and my IDE is not complaining